library-best-practices/JLBP-6.md

```diff
@@ -87,14 +88,14 @@ Given this scenario, here are the possible combinations of renamings:
     for users to ensure that their build tree only includes one of the
     artifacts. **NEVER DO THIS**.
 - **Rename Java package**:
-  - **Don't rename Maven ID (case 3)**: The classes from `g1:a1:1.0.0` and
+  - **Case 3: Don't rename Maven ID**. The classes from `g1:a1:1.0.0` and
     `g1:a1:2.0.0` could technically be used together, but since they share the
     Maven group ID and artifact ID, only one jar can be pulled in. Users are
     forced to change their code if they want to add any references to the new
```

Comment:
"are forced to change their code if they want to add any references" --> must change their code to add references